### ISSUE TITLE
Procedural tanks, RO tanks, having all tank types available

### ISSUE THREAD

**Issue by gonsy00** [KSP-RO/RP-0#1721]
As I saw on the hotfix with the procedural avionics, there is the same issue with the fuel tanks type. Meaning all coper, aluminium, etc are available from the begining.

**Comment by NathanKell** [comment]
Did you actually try to *build* the craft? Because, like engines, you can now select any config to test, and validation is done at build time.

**Comment by gonsy00** [comment]
You are completly right. It does actually display a message telling you you dont have the tecnology to do it. (sorry for that)

What you can actually do is to tool the tank/avionics with a configuration that you cant use.
I undestand that gives you the option to plan in advance in the VAB, wich is cool
So the question here is, is that tooling option something intended, or is a problem after the switch?

Is there any option to, for exemple, colour the different names of tank aloys on the tank display? or the SM? 
So basically having an option to stop the tooling, or to be aware wich ones are the ones available.

Thanks for the answer

**Comment by NathanKell** [comment]
It has always been the case that you can tool locked parts--for example, you can tool avionics tech you don't have access to. So far that's working as expected. It's certainly possible tooling could be prevented on parts that fail validation; might be worth making an issue to that effect.

Closing this since the original report was not the case.

**Comment by gonsy00** [comment]
Thanks for you answers
